main.go

```diff
@@ -178,39 +170,9 @@ func cmdPush(client *distribution.Client, args []string) int {
 		fmt.Fprintf(os.Stderr, "Continuing anyway, but this may cause issues.\n")
 	}
 
-	// Parse the reference
-	ref, err := name.ParseReference(reference)
-	if err != nil {
-		fmt.Fprintf(os.Stderr, "Error parsing reference: %v\n", err)
-		return 1
-	}
-
-	// Create image with layer
-	var mdl types.ModelArtifact
-	mdl, err = gguf.NewModel(source)
-	if err != nil {
-		fmt.Fprintf(os.Stderr, "Error creating model from gguf: %v\n", err)
-		return 1
-	}
-
-	// Add all license files as layers
-	for _, path := range licensePaths {
-		fmt.Println("Adding license file:", path)
-		licenseLayer, err := partial.NewLayer(path, types.MediaTypeLicense)
-		if err != nil {
-			fmt.Fprintf(os.Stderr, "Error adding license layer for %s: %v\n", path, err)
-			return 1
-		}
-		mdl = mutate.AppendLayers(mdl, licenseLayer)
-	}
```

Comment:
Aren't we losing the ability to add licenses with the model distribution tool with this change? I think we should keep the CLI as is for now (reaching into the internal packages) until we add support for this to the client.